examples/use_plugins.py

```
@@ -0,0 +1,127 @@
+    enable_plugin,
+    get_plugin,
+    get_available_plugins,
+    get_all_plugin_metadata,
+)
+
+
+def main():
+    """Main function demonstrating plugin usage."""
+    # Add the examples/plugins directory to the plugin search path
+    plugin_dirs = [Path(__file__).parent / "plugins"]
+    plugin_manager = get_plugin_manager(plugin_dirs)
```

Ruff (0.8.2)

Local variable `plugin_manager` is assigned to but never used

Remove assignment to unused variable `plugin_manager`

(F841)